atlasdb-dbkvs/src/main/java/com/palantir/atlasdb/keyvalue/dbkvs/impl/oracle/OracleDdlTable.java

```diff
@@ -243,26 +250,80 @@ private void executeIgnoringError(String sql, String errorToIgnore) {
     }
 
     @Override
-    public void compactInternally() {
+    public void compactInternally(boolean inSafeHours) {
+        final String compactionFailureTemplate = "Tried to clean up {} bloat after a sweep operation,"
```

Comment:
Minor: this message is a little out of date.  e.g. Assumes compact is running after a sweep, also not sure what "Since this can't be automated in your configuration," means and whether it is still correct.

Reply:
Fixed the message on compact after sweep.

Regarding "since this can't be automated", I suspect the writer was referring to a world in which we didn't have safe hours (and so running SHRINK SPACE all the time may not have been safe)?